.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  release:
+    types: [released]
```

Comment:
If I'm not wrong,  the release script right now, creates the github release and on a second API call uploads the zip right? How do you ensure that this triggers only after the zip upload?

Reply:
Ah, that's interesting, I hadn't considered that. I think we can accomodate to that by setting the release to draft status when first creating it, and only removing that status after the release asset has been attached.

Reply:
I think I can also include an extra guard to make sure the workflow isn't run unless there's at least one asset attached to the release :thinking:

Reply:
> Ah, that's interesting, I hadn't considered that. I think we can accomodate to that by setting the release to draft status when first creating it, and only removing that status after the release asset has been attached.

Addressed by the latest two commit, and spun off into a separate PR that can be reviewed separately: #27713

Reply:
> I think I can also include an extra guard to make sure the workflow isn't run unless there's at least one asset attached to the release :thinking:

Added that criterion now :+1:

Reply:
> Addressed by the latest two commits, and spun off into a separate PR that can be reviewed separately: #27713

Merged that PR, and rebased this one on `master`.